app/src/main/webapp/app/error/error.ctrl.js

```diff
@@ -170,5 +170,54 @@ esbMessageAdminApp.controller('ErrorsSearchCtrl', [
       $scope.calendarToOpened = true;
     };
     // End Date picker stuff
+
+    // on message select, fetch message details
+    $rootScope.$watch('selectedMessage',
```

Comment:
We can change this to just `$scope` instead of `$rootScope` now, and then on line 140 and 142 we can (and must) similarly change this to use `$scope` instead of `$rootScope` as well.

You may have to make some minor test updates after this